ci/test/00_setup_env_s390x.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+export HOST=s390x-unknown-linux-gnu
```

Comment:
```suggestion
export HOST=s390x-unknown-linux-gnu
# The host arch is unknown, so we run the tests through qemu.
# If the host is arm and wants to run the tests natively, it can set QEMU_USER_CMD to the empty string.
export QEMU_USER_CMD="${QEMU_USER_CMD:"qemu-s390x"}"
```

And then add in the travis.yml:

```
        QEMU_USER_CMD=""  # Can run the tests natively without qemu
```

(just like we do for arm)

Reply:
So should I also add the QEMU packages that we have in the ARM script?

Reply:
Ah good catch, yes.